crates/commonware-node/src/dkg/ceremony/persisted.rs

```diff
@@ -57,12 +57,19 @@ impl Read for State {
         _cfg: &Self::Cfg,
     ) -> Result<Self, commonware_codec::Error> {
         let num_players = UInt::read_cfg(buf, &())?.into();
-        let dealing = Option::<Dealing>::read_cfg(buf, &(quorum(num_players as u32) as usize))?;
+        if num_players == 0 {
```

Comment:
nice

Reply:
Yeah, this is a logic change, but I thought it wouldn't so bad to sneak this in here. :-)